src/ui/linkformview.js

```diff
@@ -210,6 +248,84 @@ export default class LinkFormView extends View {
 
 		return button;
 	}
+
+	/**
+	 * Populates {@link module:ui/viewcollection~ViewCollection} of {@link module:ui/button/switchbuttonview~SwitchButtonView}
+	 * made based on {@link #manualDecorators}
+	 *
+	 * @private
+	 * @returns {module:ui/viewcollection~ViewCollection} of Switch Buttons.
+	 */
+	_createManualDecoratorsUIView() {
```

Comment:
It does not create a view. It creates a collection (of switches), which should be clear in the name of the method.